internal/grid/connection.go

```diff
@@ -206,8 +206,8 @@ func newConnection(o connectionParams) *Connection {
 		Local:              o.local,
 		id:                 o.id,
 		ctx:                o.ctx,
-		outgoing:           xsync.NewMapOfPresized[uint64, *muxClient](1000),
-		inStream:           xsync.NewMapOfPresized[uint64, *muxServer](1000),
+		outgoing:           xsync.NewMapOfPresized[uint64, *muxClient](2000),
+		inStream:           xsync.NewMapOfPresized[uint64, *muxServer](2000),
```

Comment:
```suggestion
		outgoing:           xsync.NewMapOfPresized[uint64, *muxClient](1000),
		inStream:           xsync.NewMapOfPresized[uint64, *muxServer](1000),
```

Again a per connection property - and just a pre-alloc anyway.